addon/components/variable-plugin/autofilled/nodeview.hbs

```diff
@@ -0,0 +1,25 @@
+{{! @glint-nocheck: not typesafe yet }}
+<AuPill
+  @icon={{this.PencilIcon}}
+  @iconAlignment='right'
+  class='variable'
+  {{on 'click' this.onClick}}
+>
+  <EmberNode::EmbeddedEditor
```

Comment:
Do we actually need an embedded editor? As it is you can actually replace the content with text, which does correctly get replaced with the value, but it still seems confusing to the user.

Reply:
Fixed that, thank you for spotting the problem :)

Reply:
It seems like you understood something different than I meant from my comment, but I'm glad it helped spot a bug.

What I meant was, that I don't see why we want an embedded editor here at all. To my mind, the contents of the variable should only be changeable by setting a different key or label. Here's a screenshot to demonstrate what I mean. This is a document with 2 autofill variables. You can see that I typed over the contents of the second, which means the only way to know that it's an autofill variable is to look in the sidebar. Is there some reason that template builders would want to be able to do this that I am missing?

![image](https://github.com/user-attachments/assets/d6752369-84d4-44f4-b02a-54285fb35660)